udata/core/reuse/models.py

```diff
@@ -45,7 +45,8 @@ class Reuse(db.Datetimed, WithMetrics, BadgeMixin, db.Owned, db.Document):
     title = db.StringField(required=True)
     slug = db.SlugField(
         max_length=255, required=True, populate_from='title', update=True)
-    description = db.StringField(required=True)
+    description = db.MDStripField(required=True, populate='description_md')
+    description_md = db.StringField()
```

Comment:
Why switch the field ? It breaks the API (we need to keep the unrendered fields at the same place).
**Proposal**: simply add:
```python
description_rendered = db.MDStripField(required=True, populate='description')
```